src/pages/ManageComputeHost/ActivateComputeHost.js

```diff
@@ -106,6 +107,13 @@ class ActivateComputeHost extends BaseUpdateWizardPage {
     this.setState(state);
   }
 
+  setCloseButtonDisabled = () => {
```

Comment:
I see you have the same `setCloseButtonDisabled` set five times across 5 different components. Would it be better to move this up to `BaseUpdateWizardPage` or another higher level component? If this function has to change, it could result in a problem if one of these duplicates is missed.

Reply:
Discussed with @dotNomad  will make a common function in BaseUpdateWizardPage, child page will still need to overwrite the setCloseButtonDisabled default behavior.

Reply:
found a simpler way to just implement in BaseUpdateWizardPage to take care of all the cases.